### PR TITLE
feat: support dataloader with multiple inputs

### LINKED ISSUE #75: DataLoader with multiple inputs caused errors
I have a train_loader with multiple inputs:

train_loader = torch.utils.data.DataLoader( torch.utils.data.TensorDataset(input1, input2, input3, label) ,batch_size=batch_size, shuffle=True)

These inputs are arrays with different shapes, so it's hard to concatenate them into a single tensor. They work well with normal pytorch code. However, Ensemble-Pytorch can not deal with it.
Hope Ensemble-Pytorch can support DataLoader with multiple inputs in the future.
Thank you.

### PR DESCRIPTION
**Pull request by xuyxu**
resolves #75

#### Steps
* [x] code: make existing ensembles adapt to multiple inputs using non-keyword arguments
* [x] test: add unit tests on the multiple inputs case for all ensembles

### CODE REVIEW

**Review by xuyxu — COMMENTED** [review]
LGTM.
